vdr/vdr.go

```diff
@@ -124,13 +124,56 @@ func (r *VDR) ConflictedDocuments() ([]did.Document, []types.DocumentMetadata, e
 // Diagnostics returns the diagnostics for this engine
 func (r *VDR) Diagnostics() []core.DiagnosticResult {
 	// return # conflicted docs
-	count, _ := r.store.ConflictedCount()
+	totalCount := 0
+	ownedCount := 0
+
+	// uses dedicated storage shelf for conflicted docs, does not loop over all documents
+	err := r.store.Conflicted(func(doc did.Document, metadata types.DocumentMetadata) error {
+		totalCount++
+		controllers, err := r.didDocResolver.ResolveControllers(doc, &types.ResolveMetadata{Hash: &metadata.Hash})
+		if err != nil {
+			return err
```

Comment:
I think this causes the iterator to stop at this point, causing subsequent conflicted documents not to be addressed?

Reply:
Changed to log and return nil (continue)